mempool/src/shared_mempool/types.rs

```diff
@@ -235,8 +236,13 @@ pub type SubmissionStatus = (MempoolStatus, Option<DiscardedVMStatus>);
 pub type SubmissionStatusBundle = (SignedTransaction, SubmissionStatus);
 
 pub enum MempoolClientRequest {
+    /// Submits a transaction to the mempool and returns its submission status
     SubmitTransaction(SignedTransaction, oneshot::Sender<Result<SubmissionStatus>>),
+    /// Retrieves a signed transaction from the mempool using its hash
     GetTransactionByHash(HashValue, oneshot::Sender<Option<SignedTransaction>>),
+    /// Retrieves all addresses with transactions in the mempool's parking lot and
+    /// the number of transactions for each address
+    GetAddressesFromParkingLot((), oneshot::Sender<Vec<(AccountAddress, usize)>>),
```

Comment:
Why an empty first argument? i.e., `()`?

Reply:
Ah I just wanted to keep the pattern the same as the other enum variants with two arguments but I can get rid of it